packages/ui/src/molecules/PaymentMethods/PaymentMethods.tsx

```diff
@@ -0,0 +1,42 @@
+import type { ReactNode } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface PaymentMethodsProps {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress,
+   * testing-library, and jest).
+   */
+  testId?: string
+  /**
+   * Title of the payment methods section (e.g.: "Payment methods",
+   * "Accepted Cards").
+   */
+  title?: ReactNode
+  /**
+   * Children will receive the flags to be displayed in the payment
+   * methods section (e.g.:, visa, mastercard, etc).
+   */
+  children: ReactNode
+}
+
+const PaymentMethods = forwardRef<HTMLDivElement, PaymentMethodsProps>(
+  function PaymentMethods(
+    { testId = 'store-payment-methods', title, children },
+    ref
+  ) {
+    return (
+      <div ref={ref} data-store-payment-methods data-testid={testId}>
```

Comment:
Just something minor I've observed that could help give more flexibility for our clients: allow the wrapper `div` to receive `...otherProps` so clients can decide if they want to add a CSS class via `className` or a different `role`.

```suggestion
    { testId = 'store-payment-methods', title, children, ...otherProps },
    ref
  ) {
    return (
      <div ref={ref} data-store-payment-methods data-testid={testId} {...otherProps}>
```

Note that the formatting for my above's suggestion may conflict with Prettier, so do it on VSCode.

Reply:
Nice catches, @filipewl. 
I just bump into some of these approaches in  #1031 as well. 
The 2 most important points that you mentioned above make total sense to me, the others are something we can discuss in the future.